sql/expression/aggregation.go

```diff
@@ -107,3 +107,57 @@ func (e *First) Merge(buffer, partial sql.Row) {
 func (e *First) Eval(buffer sql.Row) interface{} {
 	return buffer[0]
 }
+
+// Min aggregation returns the smallest value of the selected column.
+// It implements the AggregationExpression interface
+type Min struct {
+	UnaryExpression
+}
+
+func NewMin(e sql.Expression) *Min {
```

Comment:
the rest of the exported members are missing comments

Reply:
Done